lumicks/pylake/calibration.py

```diff
@@ -1,3 +1,218 @@
+    mapping = {
+        "bead_diameter": "Bead diameter (um)",
+        "rho_bead": "Bead density (Kg/m3)",
+        "rho_sample": "Fluid density (Kg/m3)",
+        "viscosity": "Viscosity (Pa*s)",
+        "temperature": "Temperature (C)",
+        "driving_frequency_guess": "Driving data frequency (Hz)",
+        "distance_to_surface": "Bead center height (um)",
+        # Fixed diode parameters (only available when diode was fixed)
+        "fixed_alpha": "Diode alpha",
+        "fixed_diode": "Diode frequency (Hz)",
+        # Only available for axial with active
+        "drag": "gamma_ex_lateral (kg/s)",
+    }
+
+    return {key_param: bl_dict.get(key_bl) for key_param, key_bl in mapping.items()}
+
+
+class ForceCalibrationItem(UserDict):
+    def _verify_full(self):
+        if (kind := self.data.get("Kind", "Unknown")) != "Full calibration":
+            raise ValueError(
+                "These parameters are only available for a full calibration. Instead, this is a "
+                f"calibration item where the following operation was applied: {kind}."
+            )
+
+    def power_spectrum_params(self):
+        """Returns parameters with which the power spectrum was calculated
+
+        Examples
+        --------
+        ::
+
+            import lumicks.pylake as lk
+
+            f = lk.File("passive_calibration.h5")
+            calibration = f.force1x.calibration[1]  # Grab a calibration item for force 1x
+
+            # Slice the data corresponding to the calibration we want to reproduce.
+            calib_slice = f.force1x[calibration]
+
+            # De-calibrate to volts using the calibration that was active before this slice.
+            previous_calibration = calib_slice.calibration[0]
+            calib_slice = calib_slice / previous_calibration.force_sensitivity
+
+            power_spectrum_params = previous_calibration.power_spectrum_params()
+            power_spectrum = lk.calculate_power_spectrum(calib_slice.data, **power_spectrum_params)
+            power_spectrum.plot()
+        """
+        self._verify_full()
+
+        return {
+            "num_points_per_block": self.num_points_per_block,
+            "sample_rate": self.sample_rate,
+            "excluded_ranges": self.excluded_ranges,
+            "fit_range": self.fit_range,
+        }
+
+    def _model_params(self):
+        """Returns parameters with which to create an active or passive calibration model"""
+
+        # TODO: Model needs to support fixed_diode and fixed_alpha, keep API private until finalized
+        self._verify_full()
+        model_params = _read_from_bl_dict(self.data)
+        model_params["sample_rate"] = self.sample_rate
+        model_params["fast_sensor"] = self.fast_sensor
+        model_params["axial"] = bool(self.data.get("Axial calibration"))
+        model_params["hydrodynamically_correct"] = bool(
+            self.data.get("Hydrodynamic correction enabled")
+        )
+        return model_params
```

Comment:
just a thought, but using a `dict` union could be a nice pattern to use here and elsewhere

```suggestion
        return _read_from_bl_dict(self.data) | {
            "sample_rate": self.sample_rate,
            "fast_sensor": self.fast_sensor,
            "axial": bool(self.data.get("Axial calibration")),
            "hydrodynamically_correct": bool(self.data.get("Hydrodynamic correction enabled")),
        }
```

Reply:
Yeah, I prefer that actually.